src/tsdf/constants.py

```diff
@@ -49,3 +51,33 @@
     # etc
 }
 """ List of data types that are supported within the TSDF metadata file. """
+
+METADATA_NAMING_PATTERN = "**meta.json"
+""" Naming convention for the metadata files. ** allows for any prefix, including additional directories. """
+
+
+class TestConstants:
+    """Class containing constants used for testing and demonstration."""
+
+    TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "..", "..", "tests", "data")
+
+    """ Path to the test data directory. """
+    TEST_OUTPUT_DATA_DIR = os.path.join(TEST_DATA_DIR, "outputs")
+    """ Path to the test output data directory. """
+
+    TEST_DATA_FILES = {
+        "flat": os.path.join(TEST_DATA_DIR, "flat_meta.json"),
+        "hierarchical": os.path.join(TEST_DATA_DIR, "hierarchical_meta.json"),
+        "wrongversion": os.path.join(TEST_DATA_DIR, "wrongversion_meta_fail.json"),
+        "missingkey": os.path.join(TEST_DATA_DIR, "missingkey_meta_fail.json"),
+        "example_10_3_int16": os.path.join(
+            TEST_DATA_DIR, "example_10_3_int16_meta.json"
+        ),
+        "ppp": os.path.join(TEST_DATA_DIR, "ppp_format_meta.json"),
+    }
+    """ Dictionary used for accessing test data files. """
+
+    METADATA_EXTENSION = "_meta.json"
```

Comment:
I'm not sure about this. Why not just use the full filename where required? Isn't the _meta redundant? Also, some files end with "meta_fail.json"...

Reply:
The idea was that files labelled "_meta" and "_fail" allow easier testing. 

This way we can load all the files from the test folder and test whether the "bulk import" works correctly, by specifying _meta files. That is why I introduced the "_fail" label. At the time I could not come up with a nicer naming convention. In addition, I thought that **_meta.json** makes it clear that the json files are TSDF meta files (instead of relying on naming). 

If you have an alternative in mind, please let me know.

Reply:
At the moment we are using the ***_meta.json** template, but indeed we should make the pattern more specific and test the import on a few explicitly targeted files. I originally intended the test to include new test files in the bulk import, but that might indeed cause more issues than benefits later on.